.gitignore

```diff
@@ -17,4 +17,6 @@ local_settings.py
 env.airflow
 env.harvester
 harvest_datasets_datagov_analysis.csv
-docker-compose-override.yml
\ No newline at end of file
+docker-compose-override.yml
+.env
+src/
```

Comment:
why is src/ in the .gitignore?

Reply:
Because is use to debug extensions from source when needed

Reply:
what sort of extensions? Extensions of what?

Reply:
We need the [legacy harvester extensions](https://github.com/datopian/ckan-ng-harvest/blob/66_add_CKAN_container/docker-extras/ckan/Dockerfile.dev#L16-L23) here. We are not 100% decoupled from them. Since we do not save the harvest sources outside CKAN we still going to need this.

Reply:
These are _ckan_ extensions - how are you using them here?
If you must use them - why not install them as python packages?

Reply:
removed, we now are using Viderum images